dspace-api/src/main/java/org/dspace/app/util/DCInput.java

```diff
@@ -578,4 +648,90 @@ public boolean isRelationshipField() {
     public boolean isMetadataField() {
         return isMetadataField;
     }
+
+    /**
+     * Class representing a Map of the ComplexDefinition object
+     * Class is copied from UFAL/CLARIN-DSPACE (https://github.com/ufal/clarin-dspace) and modified by
+     * @author Milan Majchrak (milan.majchrak at dataquest.sk)
+     */
+    public static class ComplexDefinitions {
+        /**
+         * Map of the ComplexDefiniton object
+         */
+        private Map<String, ComplexDefinition> definitions = null;
+        private Map<String, List<String>> valuePairs = null;
+        private static final String separator = ";";
```

Comment:
This seems a bit unfortunate; all the live data (metadatavalue rows) probably must be updated (as they contain the original separator `@@`) also for example https://github.com/ufal/clarin-dspace/pull/1025/files#diff-b0f3b5fecc512c6a47422bef799503851755ae63700631031d819622f27a6ec4R145 is not updated

Reply:
Please can you send me the name of the file and the current lines? Because that redirection shows me nothing. Thank you

Reply:
```
grep -n '@@' dspace/config/crosswalks/oai/metadataFormats/metasharev2.xsl
```

```
145:                    <xsl:value-of select="str:split(doc:metadata/doc:element[@name='local']/doc:element[@name='contact']/doc:element[@name='person']/doc:element/doc:field[@name='value'], '@@')[2]"/>
161:                    <xsl:value-of select="str:split(doc:metadata/doc:element[@name='local']/doc:element[@name='contact']/doc:element[@name='person']/doc:element/doc:field[@name='value'], '@@')[1]"/>
184:                                       <xsl:value-of select="str:split(doc:metadata/doc:element[@name='local']/doc:element[@name='contact']/doc:element[@name='person']/doc:element/doc:field[@name='value'], '@@')[4]" />
205:                            <xsl:value-of select="str:split(doc:metadata/doc:element[@name='local']/doc:element[@name='contact']/doc:element[@name='person']/doc:element/doc:field[@name='value'], '@@')[3]"/>
291:                                                            <xsl:value-of select="str:split(., '@@')[3]"/>
294:                                                            <xsl:value-of select="str:split(., '@@')[4]"/>
462:                        <xsl:value-of select="str:split(doc:metadata/doc:element[@name='local']/doc:element[@name='size']/doc:element[@name='info']/doc:element/doc:field[@name='value'], '@@')[1]"/>
479:                        <xsl:value-of select="str:split(doc:metadata/doc:element[@name='local']/doc:element[@name='size']/doc:element[@name='info']/doc:element/doc:field[@name='value'], '@@')[2]"/>
```

Reply:
Could be the solution to change `separator` in the crosswalks?